### ISSUE TITLE
Not all packages are present in the extra-alucryd repository

### ISSUE THREAD

**Issue by ilya-fedin** [alucryd/aur-alucryd#88]
```
 ilya@arch  ~  LANG=C sudo pacman -S pantheon
:: There are 19 members in group pantheon:
:: Repository community
   1) elementary-icon-theme  2) granite  3) pantheon-files  4) pantheon-music
   5) pantheon-terminal  6) pantheon-videos
:: Repository extra-alucryd
   7) capnet-assist  8) cerbere  9) contractor  10) elementary-wallpapers
   11) gtk-theme-elementary  12) lightdm-pantheon-greeter  13) pantheon-applications-menu
   14) pantheon-calendar  15) switchboard  16) switchboard-plug-printers
   17) switchboard-plug-user-accounts  18) wingpanel  19) wingpanel-indicator-session

Enter a selection (default=all): 
resolving dependencies...
warning: cannot resolve "libgala.so=0-64", a dependency of "wingpanel"
warning: cannot resolve "libwingpanel-2.0.so=0-64", a dependency of "lightdm-pantheon-greeter"
warning: cannot resolve "libgala.so=0-64", a dependency of "wingpanel"
warning: cannot resolve "libwingpanel-2.0.so=0-64", a dependency of "pantheon-applications-menu"
warning: cannot resolve "libgala.so=0-64", a dependency of "wingpanel"
warning: cannot resolve "libgala.so=0-64", a dependency of "wingpanel"
warning: cannot resolve "libwingpanel-2.0.so=0-64", a dependency of "wingpanel-indicator-session"
:: The following packages cannot be upgraded due to unresolvable dependencies:
      lightdm-pantheon-greeter  pantheon-applications-menu  wingpanel
      wingpanel-indicator-session

:: Do you want to skip the above packages for this upgrade? [y/N] 
```

**Comment by thinkiny** [comment]
+1

**Comment by alucryd** [comment]
Feel free to patch gala to work with the latest GNOME, then all packages will be in there ;)

**Comment by adam900710** [comment]
BTW, where is pantheon-session package? Is it merged with some other package?

**Comment by ilya-fedin** [comment]
> Feel free to patch gala to work with the latest GNOME, then all packages will be in there ;)

Why not provide the old GNOME in the repository?

**Comment by ilya-fedin** [comment]
@alucryd I made a package with mutter 3.28, installed it and built gala-git, it works!
[mutter328.tar.gz](https://github.com/alucryd/aur-alucryd/files/2525141/mutter328.tar.gz)

**Comment by alucryd** [comment]
Thanks, but Arch Linux is about encouraging upstream to support the bleeding edge, not be content with old deprecated versions. I'm sure upstream will work on that soon now that Juno has been released.

**Comment by ilya-fedin** [comment]
IMHO, functionality is more important.

**Comment by alucryd** [comment]
If I had already pushed gala in the official redositories, I would 100% agree. I would even have learned gala, fixed this myself, and made a pull request.

However upstream still hasn't deemed gala stable enough to make a release, and because of that I can't push half of the pantheon packages in [community], and frankly I doubt I ever will be able to.

I initially planned to bring the whole pantheon in [community] when I first applied to be a TU, now we're several years later, and we still don't have a gala release.

I and others have tried to contact upstream about this lack of release, among other things but to no avail. I guess this is my way of protesting. By not having a functional gala, I hope that arch users will take some of the pressure upstream and have them finally come forward.
